config/initializers/bigquery.rb

```diff
@@ -0,0 +1,12 @@
+require 'google/cloud/bigquery'
+
+BIG_QUERY_API_JSON_KEY = ENV['BIG_QUERY_API_JSON_KEY']
+
+if BIG_QUERY_API_JSON_KEY.present?
+  Google::Cloud::Bigquery.configure do |config|
+    config.project_id  = ENV['BIG_QUERY_PROJECT_ID']
+    config.credentials = "config/bq.json"
```

Comment:
The docs on that appear to say that this can be a JSON string of the file contents, it'll probably make sense to use an env var in deployed envs.

Reply:
:+1: Using an ENV var for JSON key file contents is the pattern used in [teaching-vacancies](https://github.com/DFE-Digital/teaching-vacancies/blob/e81f594426d731d3c88139a23b59f5cea7f48e1a/config/initializers/big_query.rb#L8)